stats.rst

```diff
@@ -0,0 +1,307 @@
+.. _stats:
+
+=================
+Stats Quick Start
+=================
+
+Flux includes an optional instrumentation facility to collect internal metrics, or "stats",
+and send them to an aggregator for eventual display in a front end graphing tool such as Graphite.
+The internal stats collection in flux is built on the `statsd <https://github.com/statsd/statsd>`_ protocol and
```

Comment:
I think we've settled on treating Flux the project as a proper noun, i.e. capitalized, vs `flux` the command with no capitalization, i.e.:

```suggestion
The internal stats collection in Flux is built on the `statsd <https://github.com/statsd/statsd>`_ protocol and
```